api/agentpb/query.go

```diff
@@ -100,6 +100,8 @@ func makeValue(value interface{}) (*QueryActionValue, error) {
 			Subtype: int32(v.Subtype),
 			Bytes:   v.Data,
 		}}}, nil
+	case primitive.ObjectID:
+		return &QueryActionValue{Kind: &QueryActionValue_Bytes{Bytes: []byte(v.String())}}, nil
```

Comment:
In what case this type is used?

Reply:
We get this type in response to getDiagnosticData against replicated db